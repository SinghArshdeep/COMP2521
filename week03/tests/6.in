100
110
120
125
134
149
154
174
111
198
199
90
40
45
55
32
14
23
49
52
19
11
8
9
77
78
79
98
59
51
27
81