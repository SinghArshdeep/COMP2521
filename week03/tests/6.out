Tree:
                 100
                 / \
                /   \
               90   110
              / \     \
             /   \    120
            40   98   / \
           / \       /   \
          /   \    111   125
         /     \           \
        /       \          134
       32       45           \
      /           \          149
     14           55           \
    / \           / \          154
   /   \         /   \           \
  11   23       /     \          174
 /     / \     49     77           \
8     /   \     \     / \          198
 \   19   27    52   /   \           \
  9             /   59   78          199
               51          \
                           79
                             \
                             81

#nodes = 32,  #leaves = 9
Infix  : 8 9 11 14 19 23 27 32 40 45 49 51 52 55 59 77 78 79 81 90 98 100 110 111 120 125 134 149 154 174 198 199 
Prefix : 100 90 40 32 14 11 8 9 23 19 27 45 55 49 52 51 77 59 78 79 81 98 110 120 111 125 134 149 154 174 198 199 
Postfix: 9 8 11 19 27 23 14 32 51 52 49 59 81 79 78 77 55 45 40 98 90 111 199 198 174 154 149 134 125 120 110 100 
ByLevel: 100 90 110 40 98 120 32 45 111 125 14 55 134 11 23 49 77 149 8 19 27 52 59 78 154 9 51 79 174 81 198 199 
